ui/js/services/jobfilters.js

```diff
@@ -407,15 +407,25 @@ treeherder.factory('thJobFilters', [
          */
         function getFieldFiltersArray() {
             const fieldFilters = [];
+            const clopt = thClassificationTypes.classificationOptions;
 
             _.each($location.search(), function (values, fieldName) {
                 if (_isFieldFilter(fieldName)) {
-                    const valArr = _toArray(values);
+                    var valArr = _toArray(values);
                     _.each(valArr, function (val) {
+                        var text = '';
```

Comment:
The new way to declare a variable that will change is with ``let`` rather than ``var``.